### PR TITLE
Modified README.md

### PR DESCRIPTION
**Pull request by koushikmln**
Fixed few README.md issues which came up due to rebase.

### CODE REVIEW

**Comment by koushikmln** [comment]
@butlerx do you need any more changes?

File: README.md [hunk 1]
```diff
@@ -50,14 +48,6 @@ or
 `http://yourserver:3000/ssh/<username>?sshpass=<password>`
 
 
-## Run on HTTPS
```
Comment (butlerx) [inline README.md:53]: could you re add the following lines and rename this too autologin

**Review by butlerx — APPROVED** [review]
sorry slow review

**Comment by koushikmln** [comment]
check and see if you need any more changes.

**Review by butlerx — APPROVED** [review]

**Pull request merged**